gpt_index/indices/knowledge_graph/base.py

```diff
@@ -121,6 +126,16 @@ def _build_index_from_documents(self, documents: Sequence[BaseDocument]) -> KG:
                 logging.debug(f"> Extracted triplets: {triplets}")
                 for triplet in triplets:
                     index_struct.upsert_triplet(triplet, n)
+
+                if index_struct.should_use_embeddings():
+                    rel_embeddings = self._embed_model._get_text_embeddings(
```

Comment:
with some recent changes, see `queue_text_for_embeddding` and `get_queued_text_embeddings` for how to get embeddings in batch (it's much faster)

Reply:
Done!